.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules/
+dist
+.vscode
```

Comment:
Please add a forward slash (/) to the folder names. Without the slashes, both files and folders that match the names will be ignored.